### PR TITLE
Upgrade selenium to fix a11y test failures

### PR DESCRIPTION
**Pull request by dan-f**
@dsjen

### CODE REVIEW

**Comment by dsjen** [comment]
:dancer: thank you!

File: requirements/test.txt [hunk 1]
```diff
@@ -16,6 +16,7 @@ pep257==0.4.1
 pep8==1.6.0
 pylint==1.5.0
 logilab-common==0.61.0
-selenium>=2.44.0
+# TODO: allow greater versions once bugfix for 2.49 is released.
```
Comment (dsjen) [inline requirements/test.txt:19]: It looks like 2.49 is out now.  See, https://github.com/SeleniumHQ/selenium/releases.

Comment (dan-f) [inline requirements/test.txt:19]: ah, but 2.49 is the buggy version

Comment (dsjen) [inline requirements/test.txt:19]: Ok, never mind then! :+1:

Comment (dan-f) [inline requirements/test.txt:19]: Well, I now see that your other PR built without failure and without this fix.... Weird. I'll try removing this commit from my open PR, and if I don't see the related failure anymore I'll just close this PR.

**Pull request closed without merging**